docs/paper.bib

```diff
@@ -0,0 +1,258 @@
+	journal      = {Bulletin of the American Meteorological Society},
+	volume       = {102},
+	number       = {4},
+	pages        = {E782--E807},
+	year         = {2021},
+	doi          = {10.1175/bams-d-19-0093.1}
+}
+@book{Griffiths:2017,
+	title	     = {Advice for automation of forecasts: a framework},
+	author		 = {Griffiths, Deryn and Jack, Harry and Foley, Michael and Ioannou, Ioanna and Liu, Maoyuan},
+	year		 = {2017},
+	publisher	 = {Bureau of Meteorology},
+	url          = {http://www.bom.gov.au/research/publications/researchreports/BRR-021.pdf},
+	doi          = {10.22499/4.0021}
+}
+@Article{Foley:2020,
+	title	 	 = {Comparison of Single-Valued Forecasts in a User-Oriented Framework},
+	author		 = {Foley, Michael and Loveday, Nicholas},
+	journal		 = {Weather and Forecasting},
+	volume		 = {35},
+	number		 = {3},
+	pages		 = {1067--1080},
+	year		 = {2020},
+	publisher	 = {American Meteorological Society},
+	doi          = {10.1175/waf-d-19-0248.1}
+}
+@Article{Taggart:2022b,
+	title	 	 = {Evaluation of point forecasts for extreme events using consistent scoring functions},
+	author		 = {Taggart, Robert},
+	journal		 = {Quarterly Journal of the Royal Meteorological Society},
+	volume		 = {148},
+	number		 = {742},
+	pages		 = {306--320},
+	year		 = {2022},
+	publisher	 = {Wiley Online Library},
+	doi          = {10.1002/qj.4206}
+}
+@Article{Taggart:2022c,
+	title		 = {Point forecasting and forecast evaluation with generalized Huber loss},
+	author		 = {Taggart, Robert},
+	journal		 = {Electronic Journal of Statistics},
+	volume		 = {16},
+	number		 = {1},
+	pages		 = {201--231},
+	year		 = {2022},
+	publisher	 = {The Institute of Mathematical Statistics and the Bernoulli Society},
+	doi          = {10.1214/21-ejs1957}
+}
+@Article{Gneiting:2011,
+	title		 = {Comparing density forecasts using threshold-and quantile-weighted scoring rules},
+	author		 = {Gneiting, Tilmann and Ranjan, Roopesh},
+	journal		 = {Journal of Business & Economic Statistics},
+	volume		 = {29},
+	number		 = {3},
+	pages		 = {411--422},
+	year		 = {2011},
+	publisher	 = {Taylor & Francis},
+	doi          = {10.1198/jbes.2010.08110}
+}
+@misc{Taggart:2022d,   
+    title    	 = {Assessing calibration when predictive distributions have discontinuities.},   
+    url 		 = {http://www.bom.gov.au/research/publications/researchreports/BRR-064.pdf},   
+    author 		 = {Taggart, Robert},   
+    year 		 = {2022},   
+    note 		 = {Accessed on September 9, 2023} 
+}
+@misc{loveday2023userfocused,
+      title={A User-Focused Approach to Evaluating Probabilistic and Categorical Forecasts}, 
+      author={Nicholas Loveday and Robert Taggart and Mohammadreza Khanarmuei},
+      year={2023},
+      eprint={2311.18258},
+      archivePrefix={arXiv},
+      primaryClass={stat.AP}
```

Comment:
This arXiv preprint also has a DOI - https://doi.org/10.48550/arXiv.2311.18258

However, I think the final article may have just been published in Weather and Forecasting (https://doi.org/10.1175/WAF-D-23-0201.1), so I am guessing better to update the in article citation and the References with this version?

Reply:
Updated in https://github.com/nci/scores/pull/398